server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -148,7 +148,10 @@ public enum MetadataVersion {
     IBP_3_1_IV0(3, "3.1", "IV0"),
 
     // Support for leader recovery for unclean leader election (KIP-704)
-    IBP_3_2_IV0(4, "3.2", "IV0");
+    IBP_3_2_IV0(4, "3.2", "IV0"),
+
+    // Removes min_version_level from the finalized version range that is written to ZooKeeper
+    IBP_3_2_IV1(5, "3.2", "IV1");
```

Comment:
Should this be 3.3-IV0 since we're on the 3.3 release now? (i.e., current build version is 3.3-SNAPSHOT)